introduction.md

```diff
@@ -0,0 +1,227 @@
+more licenses in the same vein.  They also developed technology and
+practices that made it easier to collaborate across distance,
+boundaries, and time.
+
+At some point, those collaborative practices and the licenses designed
+for sharing became known as "Free" or "Open Source" and more
+specifically identified as licenses that respect the [Open Source
+Definition](https://opensource.org/osd).  In the decades since,
+software made with these methods has taken over the world.  Every
+major consumer electronics product has at least some open source
+software inside it.  It is increasingly rare to find any electronic
+device of significant complexity that is wholly proprietary.
+
+As open open source software spreads through the technology world
+sector by sector, it has inspired open movements in adjacent fields
+and even distant domains.  In this set of modules, we will focus on
+open source software and related "digital public goods".  These are
+digital products that are freely shared in ways that invite
+permissionless copying and collaboration.
+
+The focus on DPGs necessarily excludes a variety of sectors that have
+also seen "open" progress.  Open hardware, open government, open
+science, and open organizations are all interesting applications of
+the original open source vision.  Any government agency interested in
+DPGs might also be interested in those topics.  They are not DPGs,
+though, so this set of modules will not cover them.  Instead, we focus
+on:
+
+ * Open Source Software - software that can be freely run, copied,
+   modified and distributed.
+
+ * Open Standards - these are standards that are freely available to
+   the general public.  Anybody can read them.  Typically, open
+   standards are made in ways that try to reduce or eliminate patent
+   license fees for implementors.
+
+ * Open Content - this is content in any media that may to some degree
+   be freely copied, displayed, modified or distributed, much like
+   open source software.  The most popular open content licenses are
+   the Creative Commons licenses.
+
+ * Open Data - these are datasets that may be freely copied and
+   shared.
+
+ * Open AI Models - these are machine learning models that may be
+   freely shared and are to some degree trained on open data sets and
+   based on algorithms implemented in open source software.
+
+## Major Software Licenses and License Categories
+
+At the most basic level, government agencies engaging with open source
+can start with basic knowledge of a handful of software licenses.  All
+of these licenses have in common the basic permissions to run, copy,
+modify, and distribute the software.  They each try create a
+collaborative commons and a shared pool of value.  For all that these
+licenses differ from each other, they are more alike than different.
+
+We broadly describe FOSS licenses as existing on a continuum from
+"permissive" to "restrictive".  The most permissive licenses have the
```

Comment:
replace "restrictive" with "protective", which it is more commonly found in describing this spectrum

Reply:
Or possibly "reciprocal"...?

I think I've seen "restrictive" more often than "protective", but in any case even if the former is more common, I would agree with the proposition that it's inaccurate.  In general, I've gone with "non-reciprocal" vs "reciprocal" for the terms here, as those seem most straightforwardly descriptive.

(Either way, we would still need to let people know about the "permissive" terminology, since they will encounter it in the wild.)